client/app/vps/dashboard/vps-dashboard.html

```diff
@@ -1,209 +1,474 @@
-                                   data-actions="[
-                                       $ctrl.snapshotActions.delete,
-                                       $ctrl.snapshotActions.restore,
-                                       $ctrl.snapshotActions.take,
-                                       $ctrl.snapshotActions.terminate
-                                   ]"></cui-tile-item>
-                    <cui-tile-item class="cui-tile__top-bordered"
-                                   data-term="'vps_tab_veeam' | translate"
-                                   data-description="'vps_status_disabled' | translate"
-                                   data-ng-if="!$ctrl.summary.data.veeamActivated"
-                                   data-actions="$ctrl.veeamActions.order"></cui-tile-item>
-                    <cui-tile-item class="cui-tile__top-bordered"
-                                   data-term="'vps_tab_veeam' | translate"
-                                   data-description="'vps_status_enabled' | translate"
-                                   data-ng-if="$ctrl.summary.data.veeamActivated"
-                                   data-actions="[$ctrl.veeamActions.manage, $ctrl.veeamActions.terminate]"></cui-tile-item>
-                    <cui-tile-item class="cui-tile__top-bordered"
-                                   data-term="'vps_tab_SUMMARY_additional_disk' | translate"
-                                   data-description="'vps_status_disabled' | translate"
-                                   data-ng-if="!$ctrl.hasAdditionalDisk"></cui-tile-item>
-                    <cui-tile-item class="cui-tile__top-bordered"
-                                   data-term="'vps_tab_SUMMARY_additional_disk' | translate"
-                                   data-description="'vps_status_enabled' | translate"
-                                   data-ng-if="$ctrl.hasAdditionalDisk"
-                                   data-actions="[$ctrl.actions.orderAdditionalDiskOption, $ctrl.actions.terminateAdditionalDiskOption]"></cui-tile-item>
-                    <cui-tile-item class="cui-tile__top-bordered"
-                                   data-term="'vps_order_windows_title' | translate"
-                                   data-description="'vps_status_enabled' | translate"
-                                   data-ng-if="$ctrl.summary.data.windowsActivated"
-                                   data-actions="$ctrl.actions.terminateWindows"></cui-tile-item>
-                    <cui-tile-item class="cui-tile__top-bordered"
-                                   data-term="'vps_order_windows_title' | translate"
-                                   data-description="'vps_status_disabled' | translate"
-                                   data-ng-if="!$ctrl.summary.data.windowsActivated"
-                                   data-actions="$ctrl.actions.orderWindows"></cui-tile-item>
-                </cui-tile-body>
-            </cui-tile>
+            <oui-tile 
```

Comment:
I know it's probably not the appropriate place to ask about that but are you aware that component status is until now:

- CX Design: **Partial**
- UX: **Rc**

According to the documentation I wouldn't feel comfortable of using that component.
ref: https://master.ui-kit.ovh/#!/oui-angular/tile

/cc @AxelPeter @whispyy @neolitec @rfougier

Reply:
Update I've got:

- CX Design: **Partial** (meaning that some UI specs is missing but could be integrated as is since it's in a finale step).
- UX: **Rc** (meaning that "Signatures should not change, only bug fixes. This component can be used.") (ref: http://localhost:3000/#!/documentation/component-status)

Then sounds good to integrate `<oui-tile>` component. Sorry for the noise.

Reply:
localhost ? The good link : https://master.ui-kit.ovh/#!/documentation/component-status

Reply:
you have the permission to update such a comment typo. thx.